docs/reference/security/authorization/built-in-roles.asciidoc

```diff
@@ -69,6 +69,16 @@ Grants full access to all features in {kib} (including Solutions) and read-only
 Grants access to manage *all* enrich indices (`.enrich-*`) and *all* operations on
 ingest pipelines.
 
+[[built-in-roles-inference-admin]] `inference_admin`::
+Provides all of the privileges of the `inference_user` role and the full
+use of the {inference} APIs. Grants the `manage_inference` cluster privilege
+and read access to the `.inference` index.
+
+[[built-in-roles-inference-user]] `inference_user`::
+Grants the minimum privileges required to view {inference} configurations
+and perform inference. Grants the `monintor_inference` cluster privilege
+and read access to the `.inference` index.
```

Comment:
These docs need updating to match removal of the index privileges.

Reply:
Thanks, the references to the `.inference` index have been removed